scripts/scripts.js

```diff
@@ -40,19 +24,6 @@ async function loadFonts() {
   }
 }
 
-/**
- * Builds all synthetic blocks in a container element.
- * @param {Element} main The container element
- */
-function buildAutoBlocks(main) {
```

Comment:
I think, its a good idea to comment out the method definition, invocation and export. This will allow us to reuse it when required.